src/Modal/Modal.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'preact';
+import PropTypes from 'prop-types';
+import { StyledModal } from './styles';
+import ModalPortal from './ModalPortal';
+
+/**
+ * Description
+ *
+ * @example ./../../docs/components/Modal.md
+ */
+class Modal extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            content: false,
+        };
+    }
+
+    static propTypes = {
+        /**
+         * Custom styles
+         */
+        style: PropTypes.object,
+    };
+
+    static defaultProps = {};
+
+    static contextTypes = {
+        theme: PropTypes.object,
+    };
+
+    show = content => {
+        this.setState({ content });
+    };
+
+    hide = () => {
+        this.setState({ content: false });
+    };
+
+    render() {
+        const { style = {}, className } = this.props;
```

Comment:
you get props as part of the render

Reply:
```
render({ style = {}, className }) {


```

Reply:
That makes sense, and I am happy to make the change, but I tried to closely follow the [style of `Notify`](https://github.com/ajainvivek/preact-fluid/blob/master/src/Notify/Notify.js#L172). Should I add to the PR?

Reply:
no its fine.